scripts/stage-docs.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -e
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/common.sh"
+# Tmp fix - revert the nounset change made by common to match the expectations of this script.
+set +u
```

Comment:
do we still need this?

Reply:
No, I meant the `set +u`.

Reply:
ah right, yeah. We've got `set -Eu` in `common.sh` which if I'm remembering correctly means that if var subst fails in this script we'll error out rather than exiting, which means that our cleanup function won't run because it'll skip the trap. I think we probably want to leave this here for that reason.